assets/js/blocks/attribute-filter/editor.scss

```diff
@@ -9,7 +9,7 @@
 	.components-placeholder__instructions {
 		border-bottom: 1px solid #e0e2e6;
 		width: 100%;
-		padding-bottom: 1em;
+		padding-bottom: 1em !important;
```

Comment:
It's generally a good idea to avoid `!important` statements. Was this the only way to make this work consistently? Isn't there a way to increase specificity in this CSS selector, or by adding a class within the component render function?

Reply:
Thank you for reviewing it @sunyatasattva. Yeah, I agree and don't like the idea of using !important as well, but as commented in the commit message, the Placeholder component comes from the @wordpress/components library and that component already has a style applied to it, so even if we add styles to our own component, it is going to be overridden by the library.

I think the only way to avoid using !important would be if the library didn't have any style applied to it or if it allowed us to modify styles for the inner components (like for the instructions inside the Placeholder and not only for the Placeholder component itself)

Do you see another way to solve this?

Reply:
Great commit message, by the way! I had missed this as people here aren't usually as descriptive, I love it. May I suggest that you make sure the bulk of the information in the commit message are also in the PR description? It's useful when navigating and reviewing PRs.

Also, regarding the “Fixes” there should be the number of the issue after that: such as “Fixes #6739”, in this way GitHub will link the issues in the UI and also close the issue automatically when this PR is merged.

Now, regarding your points: as mentioned in the issue, it seems that the “Featured Product” (among others) as pretty much the same UI and should use the same components, how come it does have the correct margin? I'd start from there. I quickly looked at it, and it seems that this block has an extra `legend` element. Maybe this element should also be in the other components and they are doing it wrong? But it seems the ones without the `legend` element do better.

Other solutions would be wrapping the Placeholder element into another `div` and style that one? Or something like this.

Reply:
Thank you @sunyatasattva! I'm going to move the information from the commit message to the PR description

> Also, regarding the “Fixes” there should be the number of the issue after that: such as “Fixes #6739”, in this way GitHub will link the issues in the UI and also close the issue automatically when this PR is merged.

I just added a number there, related to the issue (I had manually linked both before 😅 ), so I think it is linked correctly now.

> I quickly looked at it, and it seems that this block has an extra legend element. Maybe this element should also be in the other components and they are doing it wrong? But it seems the ones without the legend element do better.

I investigated as you asked and now I'm following the same strategy that was done by the Feature Product block, so we have a div wrapping the content and that we have control and can apply the styles as needed :D